.github/workflows/reusable_build_packages.yaml

```diff
@@ -0,0 +1,164 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: x86_64 or aarch64
+        required: true
+        type: string
+    outputs:
+      version:
+        description: 'Falco version extracted from config_falco.h'
+        value: ${{ jobs.build-packages.outputs.version }}    
+
+jobs:
+  build-modern-bpf-skeleton:
+    # See https://github.com/actions/runner/issues/409#issuecomment-1158849936
+    runs-on: ${{ (inputs.arch == "aarch64") && fromJSON('[ "self-hosted", "linux", "ARM64" ]') || 'ubuntu-latest' }}
+    container: fedora:latest
+    steps:
+      - name: Install build dependencies
+        run: |
```

Comment:
Always install deps before invoking `checkout` action, to properly perform a full clone.

Reply:
(same goes for other jobs that invoke `cmake` ie: `build-packages` and `build-musl-package`.

Reply:
This is needed to actually get correct Falco version in cmake configure step, instead of `0.0.0`.